node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,9 +137,19 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType ? widgetData.previews.targetType.split(';')[0] : false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
```

Comment:
A comment from last time was missed here: 

```
When the link points to an image, wouldn't we be better off embedding it as an image directly instead of embedding it to an iframe?
```